assets/model_monitoring/components/tests/e2e/resources/uri_folder_genai_raw_log_with_events/2024/04/11/18/4fda2efaa7364394a89c37989af9db93000000.jsonl

```diff
@@ -1,6 +1,6 @@
+{"specversion":"1.0","id":"f6d983eb-2907-41ed-9b3d-57ee0a9beb13","source":"/subscriptions/1aefdc5e-3a7c-4d71-a9f9-f5d3b03be19a/resourceGroups/yuachengtestrg/providers/Microsoft.MachineLearningServices/workspaces/ai-proj-eastus/onlineEndpoints/clone-vivi-log-test/deployments/clone-vivi-log-test-1","type":"azureml.inference.app_traces","datacontenttype":"application/json","time":"2024-04-11T18:06:42Z","data":[{"name":"openai_embeddings","context":{"trace_id":"0x5aeacbad618344249578ea5fa754fe21","span_id":"0x75da2a944b3bd369","trace_state":"[]"},"kind":"SpanKind.INTERNAL","parent_id":"0xb780320a56c650fd","start_time":"2024-04-11T18:06:38.318784Z","end_time":"2024-04-11T18:06:39.240647Z","status":{"status_code":"OK"},"attributes":{"framework":"promptflow","span_type":"Embedding","function":"openai.resources.embeddings.Embeddings.create","node_name":"lookup","request_id":"a9b89bec-4623-447f-bc30-8da8a29e1145","line_run_id":"a9b89bec-4623-447f-bc30-8da8a29e1145","inputs":"{\n  \"input\": [\n    [\n      6854,\n      499,\n      10552,\n      1148,\n      362,\n      2735,\n      374,\n      30\n    ]\n  ],\n  \"model\": \"text-embedding-ada-002\"\n}","llm.response.model":"ada","embedding.embeddings":"[\n  {\n    \"embedding.vector\": \"\u003c1536 dimensional vector\u003e\",\n    \"embedding.text\": \"\u003c8 dimensional token\u003e\"\n  }\n]","__computed__.cumulative_token_count.prompt":8,"__computed__.cumulative_token_count.total":8,"llm.usage.prompt_tokens":8,"llm.usage.total_tokens":8,"output":"{\n  \"data\": [\n    {\n      \"embedding\": [\n        -0.006648039445281029,\n        -0.007914023473858833,\n        0.009004717692732811,\n      \"object\": \"embedding\"\n    }\n  ],\n  \"model\": \"ada\",\n  \"object\": \"list\",\n  \"usage\": {\n    \"prompt_tokens\": 8,\n    \"total_tokens\": 8\n  }\n}"}}],"links":[],"resource":{"attributes":{"service.name":"promptflow"},"schema_url":""}}],"collectdatatype":"pandas.core.frame.DataFrame","agent":"azureml-ai-monitoring/0.1.0b4","contentrange":"bytes 0-98772/98773","correlationid":"91e2f387-1619-4fef-8b38-338ff378b1f0","xrequestid":"91e2f387-1619-4fef-8b38-338ff378b1f0","modelversion":"default"}
```

Comment:
seems we remove embedding vector from test data? why?

Reply:
because the file is greater than 1MB, that pipeline assets-validation / Validate (pull_request) failed for big file. I have to reduce the data size to less than 1MB.